prowler/providers/aws/services/cloudfront/cloudfront_distributions_use_origin_access_control/cloudfront_distributions_use_origin_access_control.metadata.json

```
@@ -0,0 +1,32 @@
+{
+  "Provider": "aws",
+  "CheckID": "cloudfront_distributions_use_origin_access_control",
+  "CheckTitle": "Check if CloudFront distributions use OAC.",
```

```suggestion
  "CheckTitle": "Check if CloudFront distributions with S3 origin use OAC.",
```
Please, update the metadata adding that is only for CloudFront distributions with S3 origins.